task2/task2/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int a = Convert.ToInt32(Console.ReadLine());
+            int b = Convert.ToInt32(Console.ReadLine());
+            int c = Convert.ToInt32(Console.ReadLine());
```

Comment:
Хотелось бы увидеть программу с дружественным интерфейсом. Прежде чем ожтидать ввода данных от пользователя, напишите сообщение, что он должен сделать
```suggestion
            Console.WriteLine("Введите длину первого ребра параллелепипеда");
            int a = Convert.ToInt32(Console.ReadLine());
            
            Console.WriteLine("Введите длину второго ребра параллелепипеда");
            int b = Convert.ToInt32(Console.ReadLine());
            
            Console.WriteLine("Введите длину третьего ребра параллелепипеда");
            int c = Convert.ToInt32(Console.ReadLine());
```